### PR TITLE
Editorial changes based on WGLC comments.

### PR DESCRIPTION
**Pull request by cdh4u**

### CODE REVIEW

File: draft-ietf-sipcore-sip-push.xml [hunk 1]
```diff
@@ -358,7 +363,7 @@ align="center"><artwork>
   ; COLON as defined in RFC 3261
   
   The format and semantics of pns-param is specific to a given
```
Comment (liviuchircu) [inline draft-ietf-sipcore-sip-push.xml]: `pns-param` -> `pn-param`

Comment (cdh4u) [reply draft-ietf-sipcore-sip-push.xml]: Thought I fixed that... Strange... Anyway, I'll fix it :) Thanks!

File: draft-ietf-sipcore-sip-push.xml [hunk 2]
```diff
@@ -318,7 +323,7 @@ align="center"><artwork>
       <section title="sip.pns Feature-Capability Indicator">
       <t>
         The sip.pns feature-capability indicator is used in a SIP 200 (OK) response
-        or in a 555 (Push Notficiation Service Not Supported) response to indicate which 
+        or in a 555 (Push Notificiation Service Not Supported) response to indicate which
```
Comment (liviuchircu) [inline draft-ietf-sipcore-sip-push.xml]: still a typo!

Comment (cdh4u) [reply draft-ietf-sipcore-sip-push.xml]: DOH! Will fix.

File: draft-ietf-sipcore-sip-push.xml [hunk 3]
```diff
@@ -38,68 +38,79 @@
...
       </t>
       <t>
-        Due to the restriction above, applications can not be awakened by non-push notification
-        traffic or internal timers. This means that a suspended SIP UA will not be able to receive 
-        an incoming SIP request (e.g., a SIP INVITE request), or to send periodic re-registration requests.
-        Also, the SIP UA will not be able to maintain connections e.g., using the SIP Outbound 
-        Mechanism <xref target="RFC5626"/>, as it requires the UA to send periodic keep-alive 
-        messages.
+        Because of the restrictions above, Session Initiation Protocol (SIP) User Agents (UAs)
+        <xref target="RFC3261"/> can not be awaken, in order to perform periodic re-registrations
```
Comment (mickeyarnold) [inline draft-ietf-sipcore-sip-push.xml]: can not be awaken -> can not be awoken

File: draft-ietf-sipcore-sip-push.xml [hunk 4]
```diff
@@ -38,68 +38,79 @@
...
-      </t> 
+        Also, without being able to use internal timers in order to awake applications, a SIP UA 
+        will not be able to maintain connections e.g., using the SIP Outbound Mechanism <xref target="RFC5626"/>, 
+        as it requires the UA to send periodic keep-alive messages.
+      </t>
+      <t>
+        This document describes how push notification mechanisms can be used to awake
+        suspended SIP UAs, in order to be able to perform re-registrations and to receive
+        incoming SIP requests for new calls. The document defines new SIP URI parameters that
+        to provide push notification information from a UA to the SIP entity (realized as a
```
Comment (mickeyarnold) [inline draft-ietf-sipcore-sip-push.xml]: defines new SIP URI parameters that to provide ->defines new SIP URI parameters that provide

File: draft-ietf-sipcore-sip-push.xml [hunk 5]
```diff
@@ -186,137 +197,161 @@ align="center"><artwork>
...
         </t>
       </section>
       <section anchor="section.proxy.rereg" title="Trigger Periodic Re-registration">
         <t>
-          If the SIP UA needs to perform periodic re-registrations, the proxy needs to have
-          information about when those re-registrations are to be performed. The proxy
-          either needs to contain the SIP registrar functionality, or the proxy needs to retrieve 
+          In order to request push notifications that will trigger a SIP UA to send a
+          re-registration REQUEST, the SIP proxy MUST have information about when a registration
+          sill expire. The proxy either needs to be a SIP registrar, or the proxy needs to retrieve
```
Comment (mickeyarnold) [inline draft-ietf-sipcore-sip-push.xml]: sill expire -> will expire

**Review by mickeyarnold — COMMENTED** [review]
A few minor editorial comments - nothing major.

**Pull request merged**